test/docgen/assemblies.ts

```diff
@@ -29,6 +29,11 @@ export class Assemblies {
     this.addAssemblies(Assemblies.AWSCDK_1_106_0);
   }
 
+  // To test a simple assembly with variadic arguments
+  public get variadicExampleTest(): reflect.Assembly {
```

Comment:
For consistent naming:

```suggestion
  public get withVariadicParameter(): reflect.Assembly {
```